src/app/admin/contacts-list/contacts-list.component.css

```diff
@@ -42,11 +42,12 @@ body {
 	background-color: #fff;
     border-bottom: 1px solid #eee;
     table-layout:fixed;
-    width: 100%;
+	width: 100%;
+	border-collapse: collapse;
 }
 .table-filter tbody tr:hover {
 	cursor: pointer;
-	background-color: #eee;
+	background-color: #fcfcfc;
```

Comment:
It makes sense to move colors to some variables files, you may need preprocessor for it